stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -75,40 +80,81 @@ protected void onPostExecute(ResponseWrapper result) {
           }
     };
 
-    public Stripe() {
-    }
+    private String defaultPublishableKey;
 
+    /**
+     * A blank constructor to set the key later.
+     */
+    public Stripe() { }
+
+    /**
+     * Constructor with publishable key.
+     *
+     * @param publishableKey the client's publishable key
+     * @throws AuthenticationException if the key is invalid
+     */
     public Stripe(String publishableKey) throws AuthenticationException {
         setDefaultPublishableKey(publishableKey);
     }
 
-    public void setDefaultPublishableKey(String publishableKey) throws AuthenticationException {
+    /**
+     * Call to create a {@link Token}.
+     *
+     * @param card the {@link Card} used for this transaction
+     * @param publishableKey the public key used for this transaction
+     * @param callback a {@link TokenCallback} to receive the result of this operation
+     */
+    public void createToken(
```

Comment:
This method was not added -- it was moved, and javadoc comments were added to it.